pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -464,14 +464,6 @@ stdenv.mkDerivation (finalAttrs: {
         url = "https://git.ffmpeg.org/gitweb/ffmpeg.git/patch/3bb00c0a420c3ce83c6fafee30270d69622ccad7";
         hash = "sha256-oZMZysBA+/gwaGEM1yvI+8wCadXWE7qLRL6Emap3b8Q=";
       }
-    ])
-    ++ (lib.optionals (lib.versionAtLeast version "7.0") [
-      {
-        # Will likely be obsolete in >7.0
-        name = "fate_avoid_dependency_on_samples";
-        url = "https://git.ffmpeg.org/gitweb/ffmpeg.git/patch/7b7b7819bd21cc92ac07f6696b0e7f26fa8f9834";
-        hash = "sha256-TKI289XqtG86Sj9s7mVYvmkjAuRXeK+2cYYEDkg6u6I=";
-      }
```

Comment:
This does not appear to be fixed upstream yet somehow?

Reply:
Result of `nixpkgs-review pr 316998` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>handbrake</li>
  </ul>
</details>
<details>
  <summary>21 packages built:</summary>
  <ul>
    <li>ffmpeg_7</li>
    <li>ffmpeg_7-full</li>
    <li>ffmpeg_7-full.bin</li>
    <li>ffmpeg_7-full.data</li>
    <li>ffmpeg_7-full.dev</li>
    <li>ffmpeg_7-full.doc</li>
    <li>ffmpeg_7-full.lib</li>
    <li>ffmpeg_7-full.man</li>
    <li>ffmpeg_7-headless</li>
    <li>ffmpeg_7-headless.bin</li>
    <li>ffmpeg_7-headless.data</li>
    <li>ffmpeg_7-headless.dev</li>
    <li>ffmpeg_7-headless.doc</li>
    <li>ffmpeg_7-headless.lib</li>
    <li>ffmpeg_7-headless.man</li>
    <li>ffmpeg_7.bin</li>
    <li>ffmpeg_7.data</li>
    <li>ffmpeg_7.dev</li>
    <li>ffmpeg_7.doc</li>
    <li>ffmpeg_7.lib</li>
    <li>ffmpeg_7.man</li>
  </ul>
</details>

Reply:
Ohhh, it's the handbrake ffmpeg. Confusingly the same name.

It's still on ABI 7.0, so let's simply keep the patch for now.

Reply:
Re-added the patch for version 7.0, fails to apply on 7.0.1